bin/webpack

```diff
@@ -7,6 +7,9 @@ require "pathname"
 ENV["BUNDLE_GEMFILE"] ||= File.expand_path("../../Gemfile",
   Pathname.new(__FILE__).realpath)
 
+p [1,
+  2]
```

Comment:
<article><p><b>[Rubocop]</b> <span>Align the elements of an array literal if they span more than one line.</span></p><details><p></p><p>11 line , 3 column in <code>bin/webpack</code></p><dl><dt>Rule</dt><dd>Layout/ArrayAlignment</dd><dt>Severity</dt><dd>convention</dd></dl></details></article>